cv_next/server/api/supabaseHelper.ts

```diff
@@ -0,0 +1,56 @@
+import 'server-only'
+
+import { createClient, SupabaseClient, PostgrestResponse } from '@supabase/supabase-js'
```

Comment:
Not sure `createClient` is the correct one as according to docs it's for static routes: https://supabase.com/docs/guides/auth/auth-helpers/nextjs#static-routes
Consider using (these)[https://supabase.com/docs/guides/auth/auth-helpers/nextjs#server-actions]